Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -1141,14 +1141,84 @@ open class NavigationMapView: UIView {
         updateRouteDurations(along: visibleRoutes)
     }
     
+    /**
+     Controls if displayed `continuousAlternative`s will also be annotated with estimated travel time delta relative to the main route.
+     
+     Callouts text font also respects `NavigationMapView.routeDurationAnnotationFontNames` property.
+     Default value is `true`.
+     */
+    public var showsContinuousAlternativeRoutesRelativeDuration: Bool = true {
```

Comment:
For consistency with `showsCongestionForAlternativeRoutes`, name this property `showsRelativeDurationOnAlternativeRoutes`.